lxd/storage/backend_lxd.go

```diff
@@ -125,11 +125,23 @@ func (b *lxdBackend) Delete(op *operations.Operation) error {
 
 // Mount mounts the storage pool.
 func (b *lxdBackend) Mount() (bool, error) {
+	unlock := lock(getPoolMountLockID(b.name))
```

Comment:
@stgraber did you have a thought about locks in the operation? CC @monstermunchkin

Reply:
So for pool Mount/Unmount, it's impossible for us to have two of those running in parallel, so we do not need locking there.